packages/stats/gbstats/bayesian/tests.py

```diff
@@ -223,3 +227,114 @@ def get_risk(mu, sigma) -> List[float]:
         risk_ctrl = float((1.0 - prob_ctrl_is_better) * mn_pos)
         risk_trt = -float(prob_ctrl_is_better * mn_neg)
         return [risk_ctrl, risk_trt]
+
+
+@dataclass
+class BanditConfig(BayesianConfig):
+    burn: int = 0
```

Comment:
Why do we need `burn` and `iteration`?

I feel like this logic is going to be handled by the back-end. We don't need updated sample weights if we are not ready to update weights, so in those cases we won't even make a call out to `gbstats` so we shouldn't need these two arguments.

Reply:
good point, removed